.ado/templates/build-rnw.yml

```diff
@@ -74,3 +74,17 @@ steps:
   - template: upload-build-logs.yml
     parameters:
       buildLogDirectory: $(BuildLogDirectory)
+
+  - powershell: |
+        $winmd2md_url = "https://github.com/asklar/winmd2md/releases/download/v0.1.10/winmd2md.exe"
+        Invoke-WebRequest -UseBasicParsing $winmd2md_url -OutFile $env:TEMP\winmd2md.exe
+        & $env:TEMP\winmd2md.exe /experimental vnext\target\x86\Debug\Microsoft.ReactNative\Microsoft.ReactNative.winmd
```

Comment:
```suggestion
        & $env:TEMP\winmd2md.exe /experimental /outputDirectory vnext\target\WinMd2MdOutput vnext\target\x86\Debug\Microsoft.ReactNative\Microsoft.ReactNative.winmd
```

Reply:
looks like my tool had a bug with outputDirectory : ) fixed in 0.0.12